libs/blocks/media/media.js

```diff
@@ -49,6 +55,47 @@ export default function init(el) {
     if (image) image.classList.add('image');
     const img = image.querySelector(':scope img');
     if (header && img?.alt === '') img.alt = header.textContent;
+
+    // lists
+    if (row.querySelector('ul')) {
+      el.querySelector('ul').classList.add('default-list');
+    }
+
+    // subcopy
+    const actionArea = row.querySelector('p.action-area');
+    if (actionArea?.nextElementSibling?.tagName === 'P') {
+      updateInnerHtml(actionArea.nextElementSibling, 'strong', '');
+      actionArea.nextElementSibling.className = 'subcopy';
+    }
+
+    // subcopy with links
+    if (actionArea?.nextElementSibling?.tagName === 'H3') {
+      actionArea.nextElementSibling.classList.remove('heading-m', 'body-xl');
+      actionArea.nextElementSibling.classList.add('heading-xs');
+      const links = row.querySelectorAll('h3.heading-xs ~ p.body-s a, h3.heading-xs ~ p.icon-area a');
+      links.forEach((link) => {
+        link.parentElement.className = 'subcopy-link';
+        link.className = 'subcopy-link';
```

Comment:
These elements `parentElement` && `link` probably shouldn't use the same class name. `subcopy-link` 
Maybe consider adding a typography class on the link. `.body-xxs` and including this in the `typography.css` global classes.